dss-api.yml

```diff
@@ -702,6 +702,73 @@ paths:
                     enum: [unhandled_exception, illegal_arguments, bundle_already_exists]
                 required:
                   - code
+    delete:
+      summary: Delete a bundle or a specific bundle version
+      description: |
+        Delete the bundle with the given UUID. This deletion is applied across replicas.
+      security:
+        - googAuth:
+            - https://www.googleapis.com/auth/userinfo.email
+      parameters:
+        - name: uuid
+          in: path
+          description: A RFC4122-compliant ID for the bundle.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: replica
+          in: query
+          description: Replica to write to.
+          required: true
+          type: string
+          enum: [aws, gcp, azure]
+        - name: version
+          in: query
+          description: Timestamp of bundle creation in RFC3339.
+          required: false
+          type: string
+          format: date-time
+        - name: json_request_body
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              creator_uid:
+                description: UID of the user issuing the deletion.
+                type: integer
+                pattern: "[A-Za-z0-9]{,50}"
```

Comment:
hm, i don't think integers use pattern.

Reply:
Good catch. Q: Since we are validating against an email, do you think I can just scrap this and log the user's email?

Reply:
That seems reasonable.